mlflow/sentence_transformers/__init__.py

```diff
@@ -46,6 +46,8 @@
 FLAVOR_NAME = "sentence_transformers"
 SENTENCE_TRANSFORMERS_DATA_PATH = "model.sentence_transformer"
 _INFERENCE_CONFIG_PATH = "inference_config"
+_LLM_INFERENCE_TASK_EMBEDDING = "llm/v1/embeddings"
+INPUT_KEY = "input"
```

Comment:
nit, and the variable currently isn't used
```suggestion
_INPUT_KEY = "input"
```

Reply:
nit: could you also rename it to `_LLM_V1_EMBEDDING_INPUT_KEY`